# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/windowwatcher/nsPIWindowWatcher.idl
# Commit: ce60426d7e31
# Full Hash: ce60426d7e3152c92c9df31e1657bee4dc928d1b
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 2b - Remove unused nsPIWindowWatcher::FindItemWithName method. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D40499
# ==============================================================================

diff -r 40b61344f0da -r ce60426d7e31 toolkit/components/windowwatcher/nsPIWindowWatcher.idl
--- a/toolkit/components/windowwatcher/nsPIWindowWatcher.idl	Thu Aug 01 18:00:32 2019 -0700
+++ b/toolkit/components/windowwatcher/nsPIWindowWatcher.idl	Fri Jun 28 12:32:49 2019 -0700
@@ -123,28 +123,5 @@
                                        in float aOpenerFullZoom,
                                        in unsigned long long aNextRemoteTabId,
                                        in boolean aForceNoOpener);
-
-  /**
-   * Find a named docshell tree item amongst all windows registered
-   * with the window watcher.  This may be a subframe in some window,
-   * for example.
-   *
-   * @param aName the name of the window.  Must not be null.
-   * @param aRequestor the tree item immediately making the request.
-   *        We should make sure to not recurse down into its findItemWithName
-   *        method.
-   * @param aOriginalRequestor the original treeitem that made the request.
-   *        Used for security checks.
-   * @return the tree item with aName as the name, or null if there
-   *         isn't one.  "Special" names, like _self, _top, etc, will be
-   *         treated specially only if aRequestor is null; in that case they
-   *         will be resolved relative to the first window the windowwatcher
-   *         knows about.
-   * @see findItemWithName methods on nsIDocShellTreeItem and
-   *      nsIDocShellTreeOwner
-   */
-  nsIDocShellTreeItem findItemWithName(in AString aName,
-                                       in nsIDocShellTreeItem aRequestor,
-                                       in nsIDocShellTreeItem aOriginalRequestor);
 };
 